### PR TITLE
DBZ-1282 -  Automate contributor check during release pipeline

### PR DESCRIPTION
**Pull request by Naros**
https://issues.jboss.org/browse/DBZ-1282

### CODE REVIEW

File: jenkins-jobs/scripts/config/FilteredNames.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Jenkins user
+artiship
```
Comment (gunnarmorling) [inline jenkins-jobs/scripts/config/FilteredNames.txt]: Why are those three filtered out? I remember at least artiship being a real person?

Comment (Naros) [reply jenkins-jobs/scripts/config/FilteredNames.txt]: If either of you know what name to resolve these aliases to we can certainly make that connection but I was unable to find a connection by checking the commit history.

Comment (gunnarmorling) [reply jenkins-jobs/scripts/config/FilteredNames.txt]: So the one I found is that s0r01ed should be Sagar Rao. About the other two, I've just asked for their names in PRs they had created. Until we hear back I'd suggest we propagate the names as is to the list. Better than omitting them altogether IMO.

Comment (Naros) [reply jenkins-jobs/scripts/config/FilteredNames.txt]: Will do, I'll update the PR accordingly.

**Comment by gunnarmorling** [comment]
Applied. Thanks a lot, @Naros; great work!

**Pull request merged**